test/testCFile.js

```diff
@@ -0,0 +1,660 @@
+        test.done();
+    },
+
+    testCFileConstructorNoFile: function(test) {
+        test.expect(1);
+
+        var cf = new CFile({
+            project: p,
+            pathName: undefined,
+            type: cft
+        });
+        test.ok(cf);
+        test.done();
+    },
+
+    testCFileMakeKey: function(test) {
+        test.expect(2);
+
+        var cf = new CFile({
+            project: p,
+            pathName: undefined,
+            type: cft
+        });
+        test.ok(cf);
+        test.equal(cf.makeKey("This is a test"), "This is a test");
+        test.done();
+    },
+
+    testCFileParseSimpleGetByKey: function(test) {
+        test.expect(5);
+
+        var cf = new CFile({
+            project: p,
+            pathName: undefined,
+            type: cft
+        });
+        test.ok(cf);
+
+        cf.parse('char *screen_share_58 = (char *)resBundle_getLocString(res_bundle, "OK");');
+
+        var set = cf.getTranslationSet();
+        test.ok(set);
+
+        var r = set.getBy({
+            reskey: "OK"
+        });
+        test.ok(r);
+
+        test.equal(r[0].getSource(), "OK");
+        test.equal(r[0].getKey(), "OK");
+
+        test.done();
+    },
+
+    testCFileParseSimpleGetBySource: function(test) {
+        test.expect(5);
+
+        var cf = new CFile({
+            project: p,
+            pathName: undefined,
+            type: cft
+        });
+        test.ok(cf);
+
+        cf.parse('resBundle_getLocString(res_bundle, "OK");');
+
+        var set = cf.getTranslationSet();
+        test.ok(set);
+
+        var r = set.getBySource("OK");
+        test.ok(r);
+        test.equal(r.getSource(), "OK");
+        test.equal(r.getKey(), "OK");
+
+        test.done();
+    },
+    
+    testCFileParseSimpleSingleQuotes: function(test) {
+        test.expect(5);
+
+        var cf = new CFile({
+            project: p,
+            pathName: undefined,
+            type: cft
+        });
+        test.ok(cf);
+        
+        cf.parse("resBundle_getLocString(res_bundle, 'OK');");
+
+        var set = cf.getTranslationSet();
+        test.ok(set);
+
+        var r = set.getBySource("OK");
+        test.ok(r);
+        test.equal(r.getSource(), "OK");
+        test.equal(r.getKey(), "OK");
+
+        test.done();
+    },
+
+    testCFileParseJSSimpleSingleQuotes: function(test) {
+        test.expect(5);
+
+        var cf = new CFile({
+            project: p,
+            pathName: undefined,
+            type: cft
+        });
+        test.ok(cf);
+        
+        cf.parse("char *screen_share_58 = (char *)resBundle_getLocString(res_bundle, 'This is a test');");
+
+        var set = cf.getTranslationSet();
+        test.ok(set);
+
+        var r = set.getBySource("This is a test");
+        test.ok(r);
+        test.equal(r.getSource(), "This is a test");
+        test.equal(r.getKey(), "This is a test");
+
+        test.done();
+    },
+
+    testCFileParseMoreComplexSingleQuotes: function(test) {
```

Comment:
Probably should not be named "SingleQuotes" when it doesn't test the single quotes